introduction/edge-functions.mdx

```diff
@@ -0,0 +1,59 @@
+# Edge functions
+Edge Functions are server-side functions that run directly within your database environment. Edge functions in SQLite Cloud ensure maximum performance and minimal latency by running functions on the same node as your database.
+
+You can write edge functions directly in the SQLite Cloud dashboard using JavaScript, TypeScript, SQL, or Python (experimental). You can [import modules](https://bun.sh/docs/runtime/autoimport) into your JavaScript and TypeScript edge functions. Module imports for Python are not currently supported.
+
+Edge functions can be called remotely over HTTP or Websockets via API, or triggered by database events via SQLite Cloud [Webhooks](https://docs.sqlitecloud.io/docs/introduction/webhooks). Each function runs in an isolated environment using the Bun runtime. 
+
+## Getting Started
+1. Navigate to the Edge Functions page from your dashboard. [add image]
+1. Under "Development", select "Edge Function", then click the "Create" button. [add image]
+1. Write and test your function.
+    1. Select the database you want to access and an api key if necessary
+    2. When you're finished, click deploy. [image - hello world function]
+
+#### Note:
+1. Functions must return a JSON-serializable object
+
+## Guides
+
+### Interacting with your Database
+Use the following global objects to access and manipulate your database using JavaScript/TypeScript or Python:
+1. In JavaScript/TypeScript: connection
+2. In Python: ???
```

Comment:
How does the user access their sqlite database in python? @Gioee @TizianoT @gionatamettifogo

Reply:
At the moment Python is not installed on the server and to be shipped in production there is a need for more test, decide which python version to install.